blog/poc3.md

```diff
@@ -0,0 +1,120 @@
+
+The viability of routing hinges on the assumption that any node (uploader/requester) can 'reach' any other node (storer). This can be solved by maintaining a specific network topology, i.e., a connectivity pattern of the overlay network called *[kademlia](https://en.wikipedia.org/wiki/Kademlia)*. Kademlia overlay guarantees the existence of a route between retriever and storer relayed through a number of peers logarithmic in network size. This flavour of overlay is called *forwarding kademlia*.
+
+As retrieve requests are relayed towards the nodes close to the chunk address, the first peer that stores the requested chunk will send the chunk data back to the requesting peer. As the delivery is relayed back towards the requesting node, forwarding peers cache the chunk content, resulting in an *auto scaling elastic cloud*: often-accessed content is replicated throughout the network decreasing its retrieval latency. Nodes are incentivised to do such opportunistic caching as they get rewarded each time they serve a chunk. The *Swarm accounting protocol* allows for an accurate peer-to-peer accounting of such data transfer and can do service for service exchange as well as off-chain payments if balance tilts towards a peer. Caching also results in *maximum resource utilisation* in as much as nodes will fill their dedicated storage space with data passing through them. If capacity is reached, least accessed chunks are purged by a garbage collection process. As a consequence, unpopular content will end up
+getting deleted. Storage insurance (to be implemented in POC4 2019) will offer users a secure guarantee to protect important content from being purged.
+
+Since for delivery, each node needs to know only the immediate peer upstream, there is no need to know who originated the request. As a consequence forwarding kademlia allows for "anonymous private browsing" as it were.
+
+The Swarm content service offers an [HTTP interface](http://swarm-guide.readthedocs.io/en/latest/apireference.html#HTTP) via a local proxy running on your local machine and connected to the rest of the swarm. This proxy is used in the browser to serve static assets (such as HTML and Javascript for dynamic content) of dapps. The HTTP API offers [six url schemes]((http://swarm-guide.readthedocs.io/en/latest/usage.html#BZZ) that dapps can use to interact with Swarm content storage.
+A [*manifest*](http://swarm-guide.readthedocs.io/en/latest/usage.html#Manifests) is a data structure describing file collections; they specify paths and corresponding content hashes allowing for URL-based content retrieval. The `bzz` URL scheme assumes that the content referenced in the domain is a manifest and renders the content entry whose path matches the one in the request path. This offers the functionality of virtual hosting of websites without dedicated webservers. Hosting on web3 is thus analogous to web2.0, with centralised hosting taken out of the equation.
```

Comment:
IMHO urls schemes are too much information here. I would just write that there is an http api for dapps to interact with Swarm content storage. The 6 schemes are not so important details